platform/darwin/src/MGLStyleLayer.h.ejs

```diff
@@ -0,0 +1,29 @@
+<%
+  const type = locals.type;
+  const layoutProperties = locals.layoutProperties;
+  const paintProperties = locals.paintProperties;
+-%>
+// This file is generated. Do not edit.
+
+#import <Mapbox/Mapbox.h>
+
+#import "MGLTypes.h"
+
+@interface MGL<%- camelize(type) %>StyleLayer : MGLStyleLayer
+
+<% if (layoutProperties.length) { -%>
+// Layout properties
```

Comment:
Better yet, “Accessing the Layout Attributes”, since we follow the Apple documentation convention of labeling each section in the documentation with a task. Since “property” has a more generic meaning in Objective-C and Swift, “attribute” is usually used to describe the kind of property that affects presentation etc. We [use “attribute” in MGLFeature](https://github.com/mapbox/mapbox-gl-native/blob/03ebae0f2ed0e5ff2c4fe284d8d3db8960ae4938/platform/darwin/src/MGLFeature.h#L86), for example.